h2o-py/h2o/explanation/_explain.py

```diff
@@ -565,7 +567,8 @@ def shap_summary_plot(
     :param colormap: colormap to use instead of the default blue to red colormap
     :param figsize: figure size; passed directly to matplotlib
     :param jitter: amount of jitter used to show the point density
-    :returns: a matplotlib figure object
+    :param save_plot_path: a path to save the plot via using mathplotlib function savefig
+    :returns: object that contains the resulting matplotlib figure (can be accessed like result.figure)
```

Comment:
`like` doesn't sound right in this context: as/using/with ? Same for all docstr below.
```suggestion
    :returns: object that contains the resulting matplotlib figure (can be accessed using result.figure)
```